File: bowling-game/bowling-game-cs/BowlingGame/05/Game.cs
namespace BowlingGame._05; 

public class Game {
  readonly int[] rolls = new int[21];
  int currentRoll;

  public void Roll(int pins) {
    rolls[currentRoll++] = pins;
  }

  public int Score() {
    int score = 0;
    for (int i = 0; i < rolls.Length; i++) {
      score += rolls[i];
    }
    return score;
  }
}

File: bowling-game/bowling-game-py/01/Game.py
class Game:
    def __init__(self):
        self.rolls = []
        self.current_roll = 0

    def roll(self, pins: int):
        self.rolls.append(pins)
        self.current_roll += 1

    def score(self):
        score = 0
        for i in range(20):
            score += self.rolls[i]
        return score
